docs/products/postgresql/howto/check-avoid-transaction-id-wraparound.rst

```diff
@@ -0,0 +1,32 @@
+Check and avoid transaction ID wraparound
+=========================================
+
+The PostgreSQL® transaction control mechanism assigns a transaction ID to every row that is modified in the database; these IDs control the visibility of that row to other concurrent transactions.
+
+The transaction ID is a 32-bit number, where 2 billion (2 thousand million) IDs are always in the "visible past" and the remainder (about 2.2 billion) are reserved for future transactions and not visible to the running transaction. To avoid a transaction wraparound and having old, existing rows invisible when more transactions are created, PostgreSQL® requires an occasional cleanup and "freezing" of old rows.
+
+Automatic or manual vacuuming
+-----------------------------
+
+You can do this manually by executing ``VACUUM FREEZE``, but the autovacuum also does this automatically once a configured number of transactions have been created since the last freeze.
```

Comment:
Since we now have this sentence alone under a new subtitle maybe we need to give a bit more background about what "this" means.

Reply:
Ooh, yes, that's well spotted. You're definitely right.

Reply:
I've amplified a bit - hope the new version makes sense.